test/metabase/driver/exasol_test.clj

```diff
@@ -206,7 +206,7 @@
   (testing "Verify that week aggregation correctly uses 'Start of week' setting"
     ; 2023-01-01 is a Sunday, 2023-01-02 is a Monday
     (mt/test-drivers #{:exasol}
-                     (mt/dataset exasol-dataset/one-timestamp-per-day
+                     (mt/dataset exasol-dataset/one-ts-per-day
```

Comment:
Long dataset name caused assertion error in Metabase, because table name was longer than 30 chars